src/Extension/Resources/config/services.yaml

```diff
@@ -2,6 +2,8 @@ parameters:
   env(UNAME_INFO): ''
   uname_info: '%env(uname:UNAME_INFO)%'
   sdk_settings: "%kernel.project_dir%/src/Extension/Resources/config/setting/settings.yaml"
+  sdk_ci_execution_default: false
```

Comment:
I'm not sure we should have a default value in the parameters. We can have it in the line below as a pure value. It is not used anywhere else. And, probably having `development` env by default is better because it covers most of cases and we don't need to have additional type.